data_tools/integration_test.py

```diff
@@ -0,0 +1,161 @@
+import json
+import subprocess
+
+import firebase_client_wrapper as fcw
+
+import sys
+
+if (len(sys.argv) != 2):
+    print("Usage python integration_test.py crypto_token")
+    cleanupFirebaseTestEnvironment()
+    exit(1)
+
+CRYPTO_TOKEN_PATH = sys.argv[1]
+fcw.init_client(CRYPTO_TOKEN_PATH)
+
+FIREBASE_PROJECT = 'coda-integration-test'
```

Comment:
@marianamarasoiu what is this used for?

Reply:
I think my initial idea was to have a completely different firebase instance for running the integration tests on, rather than the default fir-test-b0eb7. But I see that I haven't actually used this, I think realising that I can't specify which firebase project to use like that, but through the CRYPTO_TOKEN_PATH.

Happy to delete this line if we're ok using fir-test-b0eb7 for this.

Reply:
I don't think which project is determined entirely by the CRYPTO_TOKEN_PATH - so all we'd need to do to use a different project is to supply a different crypto file?